graphql.schema.json

```diff
@@ -38744,11 +39033,11 @@
           },
           {
             "name": "Comments",
-            "description": "This query returns a list of comments associated with a commentable type. The results of this query can be affected by providing an authToken header to set a user for the operation.",
+            "description": null,
```

Comment:
Think you'll want to run `npm run augment-schema`, as it looks like we're losing the descriptions defined in `definition-map.js`